packages/itwin/property-grid/src/api/ShowNullValuesPreferenceClient.ts

```diff
@@ -9,7 +9,10 @@ const PROPERTY_GRID_NAMESPACE = "PropertyGridPreferences";
 const PROPERTY_GRID_SHOWNULL_KEY = "showNullValues";
 const LOGGER_CATEGORY = "PropertyGrid";
 
-// Get showNullValues toggle from UserPreferences (corresponds to hide / show empty fields)
+/**
+ * Get showNullValues toggle from UserPreferences (corresponds to hide / show empty fields)
```

Comment:
Not a fan of these two, filed an issue to review: https://github.com/iTwin/viewer-components-react/issues/504